README.md

```diff
@@ -534,22 +534,22 @@ var myContract = assistInstance.Contract(address, abi)
 myContract.myMethod().call()
 ```
 
-### `Transaction(txObject [, callback] [, inlineCustomMsgs])`
+### `Transaction(txObjectOrHash [, callback] [, inlineCustomMsgs])`
 
 #### Parameters
 
-`txObject` - `Object`: Transaction object (**Required**)
+`txObjectOrHash` - `Object` || `String`: Transaction object or transaction hash (**Required**)
 
 `callback` - `Function`: Optional error first style callback if you don't want to use promises
 
 `inlineCustomMsgs` - `Object`: Optional notification message overrides
 
 #### Returns
 
-`Promise` or `PromiEvent` (`web3.js 1.0`)
+`Promise` or `PromiEvent` (`web3.js 1.0`) if passed a transaction object or `undefined` if passed a transaction hash
```

Comment:
we return 'undefined' if passed a tx hash? I understand there is no promise to return, but perhaps we should return something like `true` if the passed hash is valid and valid if it's not?

Reply:
Agreed. Updated.